app/pages/nonsipp/landorpropertydisposal/CompanyBuyerNamePage.scala

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages.nonsipp.landorpropertydisposal
+
+import config.Refined.{Max50, Max5000}
+import models.SchemeId.Srn
+import pages.QuestionPage
+import play.api.libs.json.JsPath
+import utils.RefinedUtils.RefinedIntOps
+
+case class CompanyBuyerNamePage(srn: Srn, landOrPropertyIndex: Max5000, disposalIndex: Max50)
+    extends QuestionPage[String] {
+
+  override def path: JsPath =
+    Paths.disposalPropertyTransaction \ toString \ landOrPropertyIndex.arrayIndex.toString \ disposalIndex.arrayIndex.toString
```

Comment:
Please double-check if the ETMP path is right, I couldn't find any specifications on the ticket.

Reply:
Cross check with mapping doc and/or the schema (in confluence) please @AlessandroCinque. It might be 'nameOfPurchaser'.

Reply:
Now it should be right 
![image](https://github.com/hmrc/pension-scheme-return-frontend/assets/43964798/060dea97-58ab-4732-b2ca-ad3acfe8daa1)